internal/exec/stages/disks/disks.go

```diff
@@ -259,6 +259,10 @@ func (s stage) createRaids(config types.Config) error {
 			args = append(args, util.DeviceAlias(string(dev)))
 		}
 
+		for _, o := range md.Options {
+			args = append(args, string(o))
```

Comment:
...but before the device arguments?

(I'm not actually sure of this, and haven't checked.)

Reply:
Haven't checked either, but there are cases with `mdadm` that the order does matter

Reply:
On Arch, `man mdadm` gives me:

```
mdadm [mode] <raiddevice> [options] <component-devices>
```

I'll move it to before the device arguments.

Reply:
Looking at the examples in `man mdadm` it looks like this is correct.